src/grid-bots/multi-vendor-multi-contract.py

```diff
@@ -7,146 +7,106 @@
 import json
 import numpy as np
 import pandas as pd
-import scipy
-import talib
 from scipy.interpolate import interp1d
-from talib import RSI
+from talib import RSI, MACD
 import asyncio
 import time
 
+TIME_LOOKUP = {
+  '1m':  60_000,
+  '5m' : 60_000 * 5,
+  '15m': 60_000 * 15,
+  '1h':  60_000 * 60,
+  '1d':  60_000 * 60 * 24,
+}
+
+# GARCH parameters
+OMEGA = 1.605025e-08
+ALPHA = 0.193613
+BETA = 0.786155
+
 class Trading(Link):
   UPDATE_SECONDS = 60
-  MULTIPLIER = 1
   SHUT_IT_DOWN = False
   GRACEFUL_SHUTDOWN = True
-  GRID_BIDS = (40, 30, 20)
-  GRID_ASKS = (60, 70, 80)
+  GRID_BIDS = (40, 30, 20) # (40, 30, 20, 10)
+  GRID_ASKS = (60, 70, 80) # (60, 70, 80, 90)
+  LOOKBACK = 150
+  LEVEL = '1m'
   SRC = 'bitmex'                         # exchange name as in the Glossary
+  MAIN_VENUE='account1'
+  VENUE_IDS = {
+    'account1' : 0000000,
+    'account2' : 0000000,
+  }
   VENUES = {
       'account1': {
         'XBTUSDT' : {
-          'sym': 'XBTUSDT',
-          'grid_size': 400000,                    # min is 1000 == 0.001 XBT
-          'candles': {},
-          'tob': (np.nan, np.nan),
-          'max_risk': 400000*10,                  # Means max 10 times the base order size
-          'current_risk': 0,
+          'order_size': 1_000 * 400,
```

Comment:
What does the 1_000 represent? I'm not too familiar with python.

Reply:
It is same a 1000 with a "pretty" separator for thousands. It is optional.